kyuubi-server/src/main/scala/org/apache/kyuubi/sql/parser/trino/KyuubiTrinoFeAstBuilder.scala

```diff
@@ -51,4 +51,9 @@ class KyuubiTrinoFeAstBuilder extends KyuubiTrinoFeBaseParserBaseVisitor[AnyRef]
   override def visitGetCatalogs(ctx: KyuubiTrinoFeBaseParser.GetCatalogsContext): KyuubiTreeNode = {
     GetCatalogs()
   }
+
+  override def visitGetTableTypes(ctx: KyuubiTrinoFeBaseParser.GetTableTypesContext)
```

Comment:
can we import `KyuubiTrinoFeBaseParser._` so all methods in this class can be inline within 100 characters

Reply:
Thanks, updated